webpack.config.js

```diff
@@ -0,0 +1,332 @@
+const path = require("path");
+// const webpack = require('webpack');
+// const HtmlWebpackPlugin = require("html-webpack-plugin");
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const src_path = "./inst/htmlwidgets/src/";
+const build_path = path.resolve(__dirname, "./inst/htmlwidgets/build");
+
+library_module = function(name) {
+  return {
+    rules: [
+      {
+        test: /\.(png|jpg|gif|svg|woff|woff2|eot|ttf|otf)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: "css/" + name + ".[ext]"
+            }
+          }
+        ]
+      },
+      {
+        test: /\.css$/,
+        use: [
+          MiniCssExtractPlugin.loader,
+          "css-loader"
+        ]
+      }
+      // {
+      //   test: /\.css$/,
+      //   use: ["style-loader", "css-loader"]
+      // }
+    ]
+  }
+}
+
+
+library_prod = function(name, filename = name, library = undefined) {
```

Comment:
Can you add some comments to this file, in case it's someone else (i.e. me) maintaining this later? :)